docs/source/frameworks/onnx.rst

```diff
@@ -0,0 +1,329 @@
+====
+ONNX
+====
+
+
+Preface
+-------
+
+In this tutorial, we will show how to train a model with PyTorch
+TensorFlow and Scikit Learn, then we will convert resulting model to
+ONNX format, use BentoML to save it and create a prediction service.
+
+.. note::
+
+   BentoML currently only support `ONNX Runtime
+   <https://onnxruntime.ai>`_ as ONNX backend. BentoML requires either
+   ``onnxruntime>=1.9`` or ``onnxruntime-gpu>=1.9`` to be installed.
+
+.. seealso::
+
+   ONNX support multiple machine learning frameworks and each
+   framework may have unique requirements/recommendations for
+   converting its own model format to ONNX format. Please refer to
+   resources listed below for in-depth knowledge of each framework.
+
+.. tab:: PyTorch
+
+   - `Quick tutorial about exporting a model from PyTorch to ONNX <https://pytorch.org/tutorials/advanced/super_resolution_with_onnxruntime.html>`_ from official PyTorch documentation.
+   - `torch.onnx <https://pytorch.org/docs/stable/onnx.html>`_ from official PyTorch documentation. Pay special attention to section **Avoiding Pitfalls**, **Limitations** and **Frequently Asked Questions**.
+
+.. tab:: TensorFlow
+
+   TODO
+
+.. tab:: Scikit Learn
+
+   TODO
+
```

Comment:
There should be a way to do a sync tab.

Reply:
currently the tabs are synced (which means if you click tensorflow, all tabs turn to tensorflow, is this what you mean?)